src/Tokenizer/Token.php

```diff
@@ -67,6 +67,17 @@ public function __construct($token)
         }
     }
 
+    public function __debugInfo()
+    {
+        return array(
+            "\0+\0content" => $this->content,
```

Comment:
i played with it, i was able to put anything here - not only "+", but literally, any content.
why "+" ? is it standardized for privates (not for public btw?) ? any ref would be nice !